app/src/pages/docs/index.module.sass

```diff
@@ -61,5 +63,16 @@
     &:nth-child(3)
       width: 60px
 
+  code
+    padding: 0 7px
+    border-radius: 3px
+    display: inline-block
+
+  div
```

Comment:
We added styles for tags `h1-h5, code, ...` on the documents page because we know how to works these elements together. But adding styles for the `div` element is unpredictable and should be deleted. Use another way.